<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioPlayer</class>
 <widget class="QMainWindow" name="AudioPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="playlistView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>461</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="currentTrack">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>10</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="addbtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/plus.png</normaloff>:/buttons/plus.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="playbtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/play.png</normaloff>:/buttons/play.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="nextbtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/skip-next.png</normaloff>:/buttons/skip-next.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="previousbtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/skip-previous.png</normaloff>:/buttons/skip-previous.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="stopbtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/stop.png</normaloff>:/buttons/stop.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="pausebtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/pause.png</normaloff>:/buttons/pause.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
